addons/hr_calendar/models/res_partner.py

```diff
@@ -0,0 +1,110 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from pytz import UTC, timezone
+from datetime import datetime
+from collections import defaultdict
+from functools import reduce
+
+from odoo import models
+
+from odoo.addons.resource.models.utils import Intervals
+
+
+class Partner(models.Model):
+    _inherit = ['res.partner']
+
+    def _get_employees_from_attendees(self, everybody=False):
+        if everybody:
+            partners = self.env['res.partner'].search([('employees_count', '>', 0)])
+        else:
+            partners = self.filtered(lambda partner: partner.employees_count > 0)
+        return {partner.id: partner.sudo().employee_ids.filtered(
+            lambda employee: employee.company_id in self.env.companies).sudo(False) for partner in partners}
```

Comment:
I don't think this filtering is needed here, because your search already returns the active employees count..  from `_search_employees_count`